tuxemon/item/item.py

```diff
@@ -154,6 +155,7 @@ def load(self, slug: str) -> None:
         self.usable_in = results["usable_in"]
         self.target = process_targets(results["target"])
         self.effects = self.parse_effects(results.get("effects", []))
+        self.battle_menu = (results["battle_menu"] if "battle_menu" in results else "")
```

Comment:
self.battle_menu = results.get("battle_menu", "")

Reply:
> self.battle_menu = results.get("battle_menu", "")

Thankyou, I've made this change now. Thanks!